domain/model/service/state_test.go

```diff
@@ -136,6 +136,19 @@ func (d *dummyState) GetModel(
 	return info, nil
 }
 
+func (d *dummyState) GetModelByName(
```

Comment:
Can you update this to mocks?

At least then all services are consistent

Reply:
Cards are made. Will tackle in the model epic. Haven't forgotten about them.